guides/common/modules/proc_disabling-tftp-for-integration.adoc

```diff
@@ -0,0 +1,32 @@
+[id="disabling-tftp-for-integration"]
+= Disabling TFTP for integration
+
+If you want to manually manage a TFTP service, you must prevent {Project} from maintaining this service on the operating system and disable orchestration to avoid errors.
+
+[NOTE]
+====
+Disabling TFTP in {Project} does not remove the related backend service on the operating system.
+====
+
+.Prerequisites
+* You are logged in to the {ProjectWebUI}.
```

Comment:
I would drop all instances of this requirement. We do not have this anywhere else. Instead, by current convention, we always start a procedure for the WebUI with `In the {ProjectWebUI}, navigate to *X* > *Y*.`.

Reply:
Since it's now back as part of the step, I removed the prereq (but I still think it should be a prereq if the entire procedure should be run in the web UI)

Reply:
Moved to https://github.com/theforeman/foreman-documentation/issues/3813